collections/ansible_collections/massopencloud/esi/roles/network/tasks/create_network.yaml

```diff
@@ -0,0 +1,21 @@
+- name: Check if network exists
+  openstack.cloud.networks_info:
+    name: "{{ network_name }}"
```

Comment:
```suggestion
    name: "{{ network_name_or_id }}"
```

I'm actually thinking that is would be enough, if a user wants to create a network, they won't give an ID.

Reply:
I do have some (minor) concerns that someone might supply an ID-like string and expect the ID to be set to that, but I'm guessing the benefits of having a consistent parameter name outweighs that!